src/main/java/cz/smarteon/loxone/Loxone.java

```diff
@@ -267,6 +268,18 @@ private void sendCommand(final ControlCommand<?> controlCommand, final boolean s
         }
     }
 
+    /**
+     * Sends command built by {@link UserCommand}
+     * @param userCommand user command
+     */
+    public void sendUserCommand(final @NotNull UserCommand<?> userCommand) {
+        if (userCommand.isHttpSupported()) {
+            loxoneHttp.get(userCommand, loxoneAuth);
```

Comment:
I don't think we want to have HTTP preference hardcoded - let's stay with websockets by default